04_setup_ironic.sh

```diff
@@ -160,7 +163,12 @@ then
   sudo podman run -d --net host --privileged --name ipa-downloader --pod ironic-pod \
      -v $IRONIC_DATA_DIR:/shared ${IRONIC_IPA_DOWNLOADER_LOCAL_IMAGE} /usr/local/bin/get-resource.sh
 
-  sudo podman wait -i 1000ms ipa-downloader
+  # Units have been introduced in 2.x
+  if printf '2.0.0\n%s\n' "$PODMAN_VERSION" | sort -V -C; then
+      sudo podman wait -i 1000ms ipa-downloader
+  else
+      sudo podman wait -i 1000 ipa-downloader
+  fi
```

Comment:
Question: do we really need to maintain the backward compatibility to a previous (major) version? Wouldn't be simpler to bump the podman min version required by dev-scripts?

Reply:
No. CentOS doesn't have a podman that understands units, and RHEL 8.3 requires them.

Reply:
Ok got it just for this command , not in general